frontend/src/app/document/entity-toggle/entity-toggle.component.html

```diff
@@ -0,0 +1,6 @@
+<button class="button" type="button" (click)="showNamedEntityDocumentation()">
+    <span class="icon">
+        <fa-icon [icon]="actionIcons.helpAlt" aria-label="help"></fa-icon>
+    </span>
+</button>
+<em iaBalloon="Show named entities for this document">Show named entities<ia-toggle (toggled)="toggleNER.emit($event)"></ia-toggle></em>
```

Comment:
Two comments:

- The tooltip seems unnecessary here, it doesn't actually add any information.
- This component is providing the label for the toggle element but that link isn't encoded in a machine-readable way. I suggested some possible implementations for that in [this comment on your earlier PR](https://github.com/UUDigitalHumanitieslab/I-analyzer/pull/1611#discussion_r1662380657).